### PR TITLE
Feature/edit help pages

### LINKED ISSUE #77: Allow admin to edit the help page
admin needs to be able to edit the help page in a similar way to what we see on the About page.

### PR DESCRIPTION
**Pull request by luisgreg99**
fixes #77

### CODE REVIEW

**Comment by straleyb** [comment]
@terrellt @zhang4952 I had a hand in this code. Greg and I did some paired programming to help with writing some tests. Please review?

File: spec/controllers/help_controller_spec.rb [hunk 1]
```diff
@@ -20,8 +18,16 @@
...
     end
   end
+
+  describe "#general" do
+    it "should render general template" do
+      get :general, id: "general_help_page"
+      expect(response).to render_template "general"
+
+    end
+  end
```
Comment (straleyb) [inline spec/controllers/help_controller_spec.rb]: May not need this test.

File: spec/routing/help_routing_spec.rb [hunk 2]
```diff
@@ -10,4 +10,7 @@
   it "should route /help/undergraduate to help#undergraduate" do
     expect(get "/help/undergraduate").to route_to "help#undergraduate"
   end
+  it "should rout /help to help#general" do
+    expect(get "/help").to route_to "help#general"
+  end
```
Comment (straleyb) [inline spec/routing/help_routing_spec.rb]: May not need this.

**Comment by terrellt** [comment]
Alright. So if these all behave the same, you don't need multiple routes.

**Comment by terrellt** [comment]
@straleyb @luisgreg99 I'd like to pair on this with you guys some time today

**Comment by straleyb** [comment]
Sure. Im good for whenever!

**Comment by luisgreg99** [comment]
Sounds good

**Pull request merged**